app/models/time_slot.rb

```diff
@@ -0,0 +1,5 @@
+
+class TimeSlot < ApplicationRecord
+    
+  end
```

Comment:
Would be good to add some validations like asserting the presence of start_time, end_time etc